js/w3c/headers.js

```diff
@@ -296,6 +296,9 @@ define(
                 conf.publishHumanDate = utils.humanDate(conf.publishDate);
                 conf.isNoTrack = $.inArray(conf.specStatus, this.noTrackStatus) >= 0;
                 conf.isRecTrack = conf.noRecTrack ? false : $.inArray(conf.specStatus, this.recTrackStatus) >= 0;
+                conf.isMemberSubmission = conf.specStatus === "Member-SUBM";
```

Comment:
That's a good question. I see "fromSubmission" in http://www.w3.org/2001/02pd/rec54 that applies to NOTE, so the intent was perhaps to classify Member and Team Submissions as Notes. It makes sense from a maturity perspective at least.